extension/js/common/core/crypto/pgp/openpgp-key.ts

```diff
@@ -441,11 +449,19 @@ export class OpenPGPKey {
     if (pubkey.type !== 'openpgp') {
       throw new UnexpectedKeyTypeError(`Key type is ${pubkey.type}, expecting OpenPGP`);
     }
-    const raw = (pubkey as unknown as { [internal]: OpenPGP.key.Key })[internal];
-    if (!raw) {
+    const opgpKey = (pubkey as unknown as { [internal]: OpenPGP.key.Key })[internal];
+    if (!opgpKey) {
       throw new Error('Object has type == "openpgp" but no internal key.');
     }
-    return raw;
+    return opgpKey;
+  }
+
+  private static extractInsecureExternalLibraryObjFromKey = (pubkey: Key) => {
```

Comment:
Instead of `Insecure` we could say `StrengthUnchecked`, and also add a link to the issue 2715